doc_admin/AdminManual.adoc

```diff
@@ -381,10 +385,16 @@ NOTE: Folder structure and known issues are found at:
 https://github.com/informatici/openhospital/blob/master/POH-README.md
 
 [#backup-restore]
-=== 3.3 Backup & Restore (alternative with MySQL Workbench®)
+=== 3.3 Backup & Restore
+
+Backup of the Open Hospital database can be performed in several ways.
+
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-s".
```

Comment:
Wasn't the decision made to remove "Starting with version xxx" and similar phrases as the document matches the software being released?

Reply:
Right, changed that

Reply:
You are right, and this is really an exception that I would keep, just because the changes are pretty big for 1.11 and I would be "conservative" in pointing out that from now on the scripts can be used for different operations.